babel/board.js

```diff
@@ -46,12 +47,48 @@ export default class Board {
         return _.uniq(_.flatten(this.orbs)).sort();
     }
 
-    /**
-     * @returns {Object} - A match event. Includes the
-     *
-     */
+    findMatches() {
+        let matches = [],
+            b = _.cloneDeep(this.orbs);
```

Comment:
I was going to ask you about cloning. In this case, I think it is just so there isn't `this.orbs` everywhere. I guess it was aesthetic here.